templates/homepage.html

```diff
@@ -4,36 +4,42 @@
 
 <h2>Create an account</h2>
 
-<form action='/new_user' method='POST'>
+    {% if not session.get('current_user') %}
+        <h2>Login</h2>
 
-    <label for='first_name'>First name:</label>
-    <input type='text' id='first_name' name='first_name'><br>
+        <form action='/login' method='POST'>
 
-    <label for='last_name'>Last name:</label>
-    <input type='text' id='last_name' name='last_name'><br>
```

Comment:
As much as possible, try to make sure that your form fields match your model property names. This makes handling them much more straight-forward and consistent.